File: hrmsFe/src/app/model/createBlackListResponse.ts
import {GetJobSeekerResponse} from "./getJobSeekerResponse";
import {GetHumanResourceEntityResponse} from "./getHumanResourceEntityResponse";

export interface CreateBlackListResponse{

  id:string,

  description:string,

  jobSeeker:GetJobSeekerResponse

  humanResourceEntity:GetHumanResourceEntityResponse
}

File: hrms/src/main/java/com/obss/hrms/response/CreateBlackListResponse.java
package com.obss.hrms.response;

import com.obss.hrms.entity.HumanResource;
import com.obss.hrms.entity.HumanResourceEntity;
import com.obss.hrms.entity.JobSeeker;

public record CreateBlackListResponse(
        String id,
        String description,
        JobSeeker jobSeeker,
        HumanResourceEntity humanResourceEntity
) {
}
